oap-server/server-bootstrap/src/main/resources/otel-oc-rules/istio-controlplane.yaml

```diff
@@ -0,0 +1,113 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This will parse a textual representation of a duration. The formats
+# accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+# with days considered to be exactly 24 hours.
+# <p>
+# Examples:
+# <pre>
+#    "PT20.345S" -- parses as "20.345 seconds"
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+defaultMetricLevel: service(['cluster', 'app'])
+metricsRules:
```

Comment:
As you mentioned in other docs, these metrics with storage entities would be auto generated, as you activate all otel-oc rules in default. Is this a good idea? My concerns are 
1. Name conflicting could happen easily, with we have incoming dolphinscheduler and shardingsphere dashboard with metrics system.
1. Unit of values could be different due to different language tradition.

Could you name these metrics starting with `istio_` for now? And consider activating the rule explicitly? What do you think?

Reply:
> as you activate all otel-oc rules in default. 
No, `otel-oc` defaults to be disabled. As I mentioned in istio metrics doc, users have to enable this receiver by hand.

But the potential conflict is a critical point that should be fixed. My preference is to fix it in “group” PR that will append the group name to metrics in a single rule file. As you mentioned, incoming dolphinscheduler and ss don't only need it, but existing agent metrics, prom, and otel also.

Reply:
What do you mean group PR? Are you going to add a new concept?

Reply:
I was thinking that PR is service names related, this is metrics name thing.

Reply:
Added `namespace` to meter metrics. now, the prefix naming rule is `meter_<namespace>_<rule_name>`

Reply:
Adding namespace would make the index name uncertain, why add this? How do we UI configuration?